lib/api/kuzzle.js

```diff
@@ -108,7 +137,7 @@ function Kuzzle () {
    *
    * @this {Kuzzle}
    */
-  this.start = function kuzzleStart () {
+  start() {
     const internalEngineBootstrap = new InternalEngineBootstrap(this, this.internalEngine);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 91 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)